src/ring_theory/polynomial/basic.lean

```diff
@@ -87,6 +87,42 @@ begin
   exact lt_of_le_of_lt (degree_X_pow_le _) (with_bot.coe_lt_coe.2 $ finset.mem_range.1 hk)
 end
 
+/-- Coefficient linear map on degree_lt -/
```

Comment:
Please make the changes in this file in a separate PR.

Reply:
#4829